spark_llm/prompt.py

```diff
@@ -131,3 +131,19 @@
 PLOT_PROMPT = PromptTemplate(
     input_variables=["columns", "explain"], template=PLOT_PROMPT_TEMPLATE
 )
+
+TEST_TEMPLATE = """
+You are an Apache Spark SQL expert, with experience writing robust test cases for PySpark code. 
+Given a PySpark function which transforms a dataframe, write a unit test class in Python with 
+methods to test the given PySpark function. 
+
+The answer must contain ONLY the code for the unit test class with the test cases (no explanation words).
+Do NOT include a main class footer (e.g. do NOT include "if __name__ == '__main__':")
+Please do include all necessary imports.
+
+Here is the function: {function}
```

Comment:
So the test is totally based on function name? Seems hacky.
Shall we make this code generation as a notebook magic instead of an API?

Reply:
{function} includes the entire function definition
and sounds good, we can convert to notebook magic